### ISSUE TITLE
Do we need DC power at all?

### ISSUE THREAD

**Issue by Orbital-Soran** [Orbital-Soran/Smarter#3]
I am not sure if we even need DC power but it is good to look into the benefits.

**Comment by Ivas-Wu** [comment]
No. Benefits literally never outweigh the inconvenience. of converting to DC, unless we wanna stick a battery in it and make it portable which is kinda big dumb.
